instagram-app/src/App.js

```diff
@@ -1,20 +1,110 @@
-import React from "react";
+import React, { Component } from "react";
 import SearchBar from "./components/SearchBar/SearchBar";
+import { Triple } from "react-preloading-component";
 
 import "./App.css";
 
 import dummyData from "./dummy-data";
 import PostContainer from "./components/PostContainer/PostContainer";
 
-function App() {
-  return (
-    <div className="App">
-      <SearchBar />
-      {dummyData.map(data => {
-        return <PostContainer key={data.username} data={data} />;
-      })}
-    </div>
-  );
-}
+export default class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      data: this.getPostsData()
```

Comment:
This return null and before it can set the item to local storage the app crash. You just need to declare this as an empty array or string to fix the problem.